HelloWorld/HelloWorld.cpp

```diff
@@ -11,18 +11,18 @@
 //    names to the corresponding #count of declarations.
 //
 // USAGE:
-//   clang -cc1 -load libHelloWorld.dylib -plugin hello-world '\'
+//   clang -cc1 -load build/lib/libHelloWorld.dylib -plugin hello-world '\'
 //    test/HelloWorld-basic.cpp
 //
 // License: The Unlicense
 //==============================================================================
-#include "clang/Frontend/FrontendPluginRegistry.h"
 #include "clang/AST/ASTConsumer.h"
 #include "clang/AST/RecursiveASTVisitor.h"
-#include "clang/Frontend/CompilerInstance.h"
-#include "llvm/Support/raw_ostream.h"
 #include "clang/Basic/FileManager.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Frontend/FrontendPluginRegistry.h"
 #include "llvm/ADT/StringMap.h"
+#include "llvm/Support/raw_ostream.h"
```

Comment:
I'm guessing that this comes from `clang-format`?

Reply:
No, I didn't use clang-format previously, It was a guess. But applying clang-format gives the same result as proposed.